### ISSUE TITLE
Making a better Neural Network

### ISSUE THREAD

**Issue by cormacmchale** [cormacmchale/KerasNeuralNetwork#4]
Now that I can train and test the Network.. I will have to research into how to better use the functions provided in Keras to build a better model

**Comment by cormacmchale** [comment]
Starting here
https://www.youtube.com/watch?v=vq2nnJ4g6N0&feature=youtu.be

**Comment by cormacmchale** [comment]
Possibly something to look at
https://adventuresinmachinelearning.com/keras-tutorial-cnn-11-lines/

**Comment by cormacmchale** [comment]
https://stackoverflow.com/questions/19766757/replacing-numpy-elements-if-condition-is-met
I think an idea could be to manipulate the training data so that it is more similar to what the web server recieves from the web page. 
Make every value either 0 or 255

**Comment by cormacmchale** [comment]
https://stackoverflow.com/questions/44959045/changing-elements-in-a-numpy-array

**Comment by cormacmchale** [comment]
https://docs.scipy.org/doc/numpy/reference/routines.array-manipulation.html

**Comment by cormacmchale** [comment]
This contains exactly what I was looking for
https://www.w3resource.com/python-exercises/numpy/python-numpy-exercise-88.php
So in code I will make all values greater than 0 = 255

**Comment by cormacmchale** [comment]
Some ideas here
https://stackoverflow.com/questions/53553797/usage-of-sigmoid-activation-function-in-keras

**Comment by cormacmchale** [comment]
I think this is esentially the approach at this point
https://www.youtube.com/watch?v=5T-iXNNiwIs

**Comment by cormacmchale** [comment]
Training again?
https://www.youtube.com/watch?v=oDHpqu52soI
Going to try this approach.. and add another layer to see if I can get better predictions

**Comment by cormacmchale** [comment]
Promising results
![results](https://user-images.githubusercontent.com/33293951/69329577-45b72d80-0c49-11ea-8b98-5088dfff75fd.PNG)

**Comment by cormacmchale** [comment]
I think now I will add the URL encode/decode option to see which works better.

**Comment by cormacmchale** [comment]
By Manipulating the data set to only have pixel values of 0,1 I have achieved the highest prediction rate for the project so far
![Capture](https://user-images.githubusercontent.com/33293951/69334799-78feba00-0c53-11ea-8f85-8c94b559e463.PNG)
Going to leave it like this for the project
